packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -131,18 +131,19 @@ const createOptions = options => {
   const currentOptGroupKeys = [];
 
   return options.map(current => {
+    const attr = current.disabled ? { disabled: true } : undefined;
     if (current.options) {
       const optGroupKeyIndex = getOptGroupKeyIndex(current.display, currentOptGroupKeys);
       const optGroupKey = getOptGroupKey(current.display, optGroupKeyIndex);
       currentOptGroupKeys.push(optGroupKey);
 
       return (
-        <optgroup key={optGroupKey} label={current.display}>
+        <optgroup {...attr} key={optGroupKey} label={current.display}>
```

Comment:
why we need const attrs for single attr...??? Can’t this be just 
```jsx
        <optgroup disabled={current.disabled} key={optGroupKey} label={current.display}>
```

Reply:
Done